crates/driver/src/infra/blockchain/gas.rs

```diff
@@ -10,26 +8,34 @@ use {
     ethcontract::dyns::DynWeb3,
     gas_estimation::{nativegasestimator::NativeGasEstimator, GasPriceEstimating},
 };
+use {crate::infra::config::file::GasEstimatorType, std::sync::Arc};
```

Comment:
Mmmh, why does `cargo fmt` not complain that these imports are not merged into one? 🤔

Reply:
Yeah, there is something weird with the gas_estimation::GasPriceEstimating, which breaks the formatter. My IDE, for example, can't resolve that type at all. Manually merged the imports.
<img width="669" alt="image" src="https://github.com/cowprotocol/services/assets/22964585/135f5e50-9549-4b21-9f5a-5948a5c00c75">